doc/contributing.rst

```diff
@@ -199,7 +199,7 @@ The two examples below might help make this process easier to understand:
 
      #. This means that the changes you made to the code changed the outputs for one or more ``rsmtool``/``rsmeval`` tests that served as inputs to one or more ``rsmcompare``/``rsmsummarize`` tests. Therefore, it is likely that the current test outputs no longer match the expected output and the tests for those two tools must be be re-run.
 
-     #. Run ``nosetests --nologcapture tests/*rsmsummarize*.py`` and ``nosetests --nologcapture tests/*rsmcompare*.py``. If you see any failures, make sure they are related to the changes you made since those are expected.
+     #. Run ``nose2 --quiet tests *rsmsummarize*.`` and ``nose2 --quiet tests *rsmcompare*.``. If you see any failures, make sure they are related to the changes you made since those are expected.
```

Comment:
```suggestion
     #. Run ``nose2 --quiet -s tests $(find tests -name 'test_*rsmsummarize*.py' | cut -d'/' -f2 | sed 's/.py//')`` and ``nose2 --quiet -s tests $(find tests -name 'test_*rsmcompare*.py' | cut -d'/' -f2 | sed 's/.py//')``. If you see any failures, make sure they are related to the changes you made since those are expected.
```